fic_modules/git.py

```diff
@@ -612,7 +633,5 @@ def extract_json_from_git(json_files, path_to_files, days_to_generate):
                                         commit_date)
             except KeyError:
                 if LOGGER:
```

Comment:
Should not we remove `if LOGGER`?
As logging will be handled at config level, not arguments.
For example: `logging.basicConfig(level=logging.INFO)`

Reply:
Removed any references for it.